pkg/utils/utils.go

```diff
@@ -35,7 +43,12 @@ func AskForData(config map[string]*project.ServiceConfig) {
 			msg := fmt.Sprintf("[%s] No ports defined to send traffic to, no provider service will be created. Do you want to create a service? y/[n]: ", svc)
 			if resp := getInput(msg); resp == "y" {
 				msg = fmt.Sprintf("[%s] Enter ports: ", svc)
```

Comment:
@kadel I am not sure where do I tell user, in what way are we expecting the ports? I feel this is the place!